docs/alert.md

```diff
@@ -0,0 +1,120 @@
+# Alert widget
+
+Provide contextual feedback messages for typical user actions with the handful of available and flexible alert messages.
+
+## Docs frameworks CSS
+
+- [Bootstrap5](https://getbootstrap.com/docs/5.0/components/alerts/)
+- [Bulma](https://bulma.io/documentation/elements/notification/)
+- [Tailwind](https://tailwindui.com/components/application-ui/feedback/alerts)
+
+## Assets
+
+To use this widget you must register the assets, you can see the documentation at [yiisoft/assets](https://github.com/yiisoft/assets#general-usage) our asset registration package, or just add an html link for asset registration, the package does not provide any `JS` code to use the close button of bulma and talwind. 
+
+## Usage
+
+Example for Bootstrap5 with icon:
+
+```php
+use Yiisoft\Yii\Widgets\Alert;
+
+echo Alert::widget()
+    ->body('An example alert with an icon')
+    ->bodyContainerClass('align-items-center d-flex')
+    ->buttonAttributes(['data-bs-dismiss' => 'alert', 'aria-label' => 'Close'])
+    ->bodyContainer(true)
+    ->buttonClass('btn-close')
+    ->buttonLabel()
+    ->class('alert alert-primary alert-dismissible fade show')
+    ->iconClass('bi bi-exclamation-triangle-fill flex-shrink-0 me-2')
+    ->id('w0-alert')
+    ->layoutBody('{icon}{body}{button}')
+    ->render(),
+```
+
+The code above generates the following HTML:
+```html
+<div id="w0-alert" class="alert alert-primary alert-dismissible fade show" role="alert">
+<div class="align-items-center d-flex">
+<div><i class="bi bi-exclamation-triangle-fill flex-shrink-0 me-2"></i></div>
+<span>An example alert with an icon</span>
+<button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+</div>
+</div>
+```
+
+Example for bulma:
+
+```php
+use Yiisoft\Yii\Widgets\Alert;
+
+echo Alert::widget()
+    ->body('An example alert.')
+    ->buttonClass('delete')
+    ->class('notification is-danger')
+    ->id('w0-alert')
+    ->layoutBody('{body}{button}')
+    ->render(),
+```
+
+The code above generates the following HTML:
+
+```html
+<div id="w0-alert" class="notification is-danger" role="alert">
```

Comment:
<div id="w0-alert" class="notification is-danger" role="alert">
    <span>An example alert.</span>
    <button type="button" class="delete">&times;</button>
</div>